tests/end_to_end/etc/cntk.json

```diff
@@ -0,0 +1,17 @@
+{
+  "jobName": "CNTK_TEST",
+  "image": "aiplatform/pai.run.cntk",
```

Comment:
>aiplatform/pai.run.cntk [](start = 12, length = 23)

where do we config the docker registry?
maybe we should put everythying in a docker image, regarding to environment configuration: docker registery, hdfs lib, cntk/tf/mxnet dependency

Reply:
read the docker registry prefix from clusterconfig.yaml?

Reply:
the docker registry in clusterconfig.yaml is for deployment. the DL framework image can be from different docker registry?

---
In reply to: [161502752](https://github.com/Microsoft/pai/pull/195#discussion_r161502752) [](ancestors = 161502752)

Reply:
yes. we can add a separate config file for end to end test later.